src/command/cheque/cashout.ts

```diff
@@ -72,7 +72,8 @@ export class Cashout extends ChequeCommand implements LeafCommand {
     const cheques = await this.getFilteredCheques(this.minimum)
     this.console.info('Found ' + cheques.length + ' cheques.')
     for (const { amount, address } of cheques) {
-      await this.cashoutOne(address, amount)
+      if (amount > 0)
+        await this.cashoutOne(address, amount)
```

Comment:
This check is not necessary.

By the time we get here in execution, the cheques are already filtered based on `minimum`, which is at least `1`.

Can you please revert this part so I can approve your other changes? Thank you.

Reply:
i'm happy to revert this, but i thought that this makes the code more resilient, i.e. if something changes/breaks with the command line parsing part...

short of a response i'll update the PR later today.

Reply:
@Cafe137 done.